<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileSetup</class>
 <widget class="QWidget" name="ProfileSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Profile Setup</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="labelSetTemp">
       <property name="text">
        <string>Set Temperature</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="setTempProfile"/>
     </item>
     <item>
      <widget class="QLabel" name="labelCoolingDelta">
       <property name="text">
        <string>Cooling Delta</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="coolDelta"/>
     </item>
     <item>
      <widget class="QLabel" name="labelHeatingDelta">
       <property name="text">
        <string>Heating Delta</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="heatDelta"/>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="Apply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
